### PR TITLE
SEO-2336 Blazor ProgressBar External Linking Changes

### PR DESCRIPTION
**Pull request by Scholar-jaruha**
Hi @ChristoIssac ,
Please find changes made on blazor progressbar external linking.
Regards,
@Scholar-jaruha

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -11,3 +11,52 @@ Make sure that you have the compatible versions of [Visual Studio 2019 (version
 ## How to run this application
 To run this application, you need to first clone the `getting-started-with-the-blazor-progressbar` repository and then open it in Visual Studio 2019. Now, simply build and run your project to view the output.
 
+## Features and Benefits
+
+## Indeterminate
+Use the indeterminate mode progress bar when progress cannot be estimated or calculated. It can be combined with determinate mode to let users know that the app is estimating the progress before the actual progress starts.
+
+## Segments
+Avoid writing several lines of code to create and position multiple progress bars: just divide a progress bar into multiple [segments](https://blazor.syncfusion.com/documentation/progress-bar/customization#segments) using a single API to visualize the progress of multiple sequential tasks.
```
Comment (Girija-Rajendran) [inline README.md:20]: add UTM param here

File: README.md [hunk 2]
```diff
@@ -11,3 +11,52 @@ Make sure that you have the compatible versions of [Visual Studio 2019 (version
...
+
+[View Demos](https://github.com/SyncfusionExamples/getting-started-with-the-blazor-progressbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Community Forums](https://www.syncfusion.com/forums/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/toolbar/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/toolbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
```
Comment (Girija-Rajendran) [inline README.md:43]: this is progressbar control right?
image

File: README.md [hunk 3]
```diff
@@ -11,3 +11,52 @@ Make sure that you have the compatible versions of [Visual Studio 2019 (version
...
+
+[Knowledge base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+
+### License
+
+This is a commercial product and requires a paid license for possession or use. Syncfusion’s licensed software, including this component, is subject to the terms and conditions of [Syncfusion's EULA](https://www.syncfusion.com/eula/es/?utm_source=github&utm_medium=listing&utm_campaign=blazor-lists-github). You can purchase a license [here](https://www.syncfusion.com/sales/products?utm_source=github&utm_medium=listing&utm_campaign=blazor-lists-github) or start a free 30-day trial [here](https://www.syncfusion.com/account/manage-trials/start-trials?utm_source=github&utm_medium=listing&utm_campaign=blazor-lists-github).
```
Comment (Girija-Rajendran) [inline README.md:50]: invalid **utm_campaign**. check and change this in all places.

File: README.md (hunk 1 above)
Comment (Scholar-jaruha) [reply README.md]: utm parameter added

File: README.md (hunk 2 above)
Comment (Scholar-jaruha) [reply README.md]: control name changed

File: README.md [hunk 4]
```diff
@@ -1,5 +1,5 @@
 # Getting Started with the Blazor ProgressBar Component
-A quick-start project that shows you get up and running with the Syncfusion ProgressBar component in a Blazor WebAssembly application. This project also contains the code snippet to configure a few of its basic features, like different types, states, and ranges.
+The [Blazor ProgressBar](https://www.syncfusion.com/blazor-components/blazor-progressbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)  is a control that indicates the progress of a task with customizable visuals. A quick-start project that shows you get up and running with the Syncfusion ProgressBar component in a Blazor WebAssembly application. This project also contains the code snippet to configure a few of its basic features, like different types, states, and ranges.
```
Comment (Girija-Rajendran) [inline README.md:2]: remove the extra space 
image

**Review by Girija-Rajendran — APPROVED** [review]

File: README.md [hunk 5]
```diff
@@ -11,3 +11,52 @@ Make sure that you have the compatible versions of [Visual Studio 2019 (version
...
+
+[sales@syncfusion.com](mailto:sales@syncfusion.com?Subject=Syncfusion Blazor ProgressBar - GitHub) | [www.syncfusion.com](https://www.syncfusion.com?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples) | Toll Free: 1-888-9 DOTNET
```
Comment (Girija-Rajendran) [inline README.md:62]: change the footer

File: README.md [hunk 6]
```diff
@@ -1,13 +1,56 @@
...
+[Suggest a feature or report a bug](https://www.syncfusion.com/feedback/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Online example](https://blazor.syncfusion.com/demos/progressbar/default-functionalities?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Video Tutorials](https://www.syncfusion.com/tutorial-videos/blazor/progressbar?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+[Knowledge base](https://www.syncfusion.com/kb/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples)
+
+### About Syncfusion Blazor Components
+The Syncfusion's [Blazor components](https://www.syncfusion.com/blazor-components?utm_source=github&utm_medium=listing&utm_campaign=blazor-progressbar-github-samples) library offers over 70 UI components to work with Blazor server-side and client-side (Blazor WebAssembly) projects seamlessly. In addition to ProgressBar, we provide popular Blazor Components such as DataGrid, Chart, Scheduler, Diagram, Word Processor, and more.
```
Comment (Girija-Rajendran) [inline README.md:44]: FT link?
image

**Pull request merged**